index.html

```diff
@@ -18,11 +18,90 @@
 
     <div class="container">
       <div class="row">
-          <div class="col-sm-12">
-              <h1>Start Here!</h1>
+          <div class="col-12">
+            <div class="jumbotron">
+              <h1 class="display-4">Natalie Schennum</h1>
+              <p class="lead">Information Technology Student</p>
+              <hr class="my-4">
+              <p>Up and Coming Software Developer</p>
+              <p class="lead">
+                <a class="btn btn-primary btn-lg" href="#" role="button">LinkedIN</a>
+              </p>
+            </div>
+          </div>
+      </div>
+      <div class="row">
+        <div class="col-4">
+          <div class="card" style="width: 18rem;">
+            <img class="card-img-top" src="https://natalieschennum.com/wp/wp-content/uploads/2018/01/Capture1-291x300.png" alt="Card image cap">
           </div>
+        </div>
+        <div class="col-12 col-lg-6">
+          
+            <div class="card-body">
+              <h5 class="card-title">About</h5>
+              <p class="card-text">Programming experience in C++, Python, Tkinter, JacaScript, HTML, and CSS. Good background in general sciences with a passion for learning, analysis, and problem solving. Looking for new insight and learning opportunities to further my knowledge base; and always happy to discuss new ideas with other developers!
+                To view some of my skills and recent projects, please follow the PORTFOLIO link below. </p>
+              <a href="#" class="btn btn-primary">PORTFOLIO</a>
+            </div>
+          
+        </div>
+        
+    </div>
+    
+    <div class="row table-row no-gutters">
+        <h4>WORK HISTORY</h4>
+        <table class="table table-striped">
+            <thead>
+              <tr>
+                <th scope="col">WHEN</th>
+                <th scope="col">WHAT</th>
+                <th scope="col">WHERE</th>
+                <th scope="col">WHY</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <th scope="row">2015-17</th>
+                <td>Molecular Biologist</td>
+                <td>University of Minnesota</td>
+                <td>Muscle Pathology Research</td>
+              </tr>
+              <tr>
+                <th scope="row">2013-14</th>
+                <td>Junior Scientist</td>
+                <td>Washington State University</td>
+                <td>Telomere Aging Research</td>
+              </tr>
+              <tr>
+                <th scope="row">2010-12</th>
+                <td>Fiduciary Assistant</td>
+                <td>East Valley Fiduciary</td>
+                <td>Client Care Coordinator</td>
+              </tr>
+            </tbody>
+          </table>
+    </div>
+   <div class="row">
+      <div class="col-12">
```

Comment:
I personally think this would look best as a nav bar at the way top. All of them seem to link to the way top. If you wanted to take a closer look at linking items with buttons, check out: http://www.hyperlinkcode.com/button-links.php

I may or may not have more resources so just let me know!